app/scripts/models/auth_brokers/base.js

```diff
@@ -205,13 +206,32 @@ define([
     },
 
     /**
-     * Is FxA account signup disabled?
+     * The default list of capabilities. Set to a capability's value to
+     * a truthy value to indicate whether it's supported.
+     */
+    defaultCapabilities: {
+      signup: true
```

Comment:
this is a great start, but it is really simplified. Not sure if that is good or bad. It totally make sense in the refactor but could be problematic for the client integration.

 In my case I wanted to make sure that a capability had more distinct names (+ versions) and were also objects that can contain extra details.